app/assets/stylesheets/settings.scss

```diff
@@ -35,6 +35,15 @@ $input-width: 650px;
       background: $light-green;
       color: #0f0f0f;
     }
+    .github-repo-fork {
```

Comment:
Maybe instead of introducing new style we could use one of the existing components called `crayons-indicator`? It's the badge we currently use, for example, as number indicator in header (notifications, connect). I imagine this could be

```
<span class="crayons-indicator crayons-indicator--accent">Fork</span>
```

It would render as something like this:

![image](https://user-images.githubusercontent.com/108287/85520466-419a2680-b603-11ea-8a74-ea3441a388e6.png)

Reply:
Hey, @ludwiczakpawel thanks for your feedback! 🤗 I can do that! But first I want to make sure we're on the same page: This PR is reverting the changes introduced in this one (#6823) in which the piece of code corresponding to the fork label styling was deleted: so no new styles are being introduced. However, it seems like those styles are a little bit outdated? So, by implementing your feedback I'll be updating them.
Having said that, should I proceed?

Reply:
correct! 👌

Reply:
Definitely will look awesome as an indicator. 🔥

Reply:
Done! 🚀